.circleci/config.yml

```diff
@@ -143,6 +143,43 @@ jobs:
                     deployment/c100-application-deployment-<< parameters.env_name >> \
                     kubernetes.io/change-cause="${BUILD_DATE} set image ${CIRCLE_PROJECT_REPONAME}:${CIRCLE_SHA1} via CircleCI"
 
+  deploy_image_to_live:
+    <<: *cloud_container
+    parameters:
+      env_name:
+        type: string
+    steps:
+      - checkout
+
+      - run:
+          name: Authenticate to LIVE cluster
+          command: |
+            echo -n ${KUBE_ENV_LIVE_STAGING_CACERT} | base64 -d > ./ca.crt
+            kubectl config set-cluster ${KUBE_ENV_LIVE_STAGING_NAME} --certificate-authority=./ca.crt --server=https://${KUBE_ENV_LIVE_STAGING_NAME}
+            kubectl config set-credentials circleci --token=${KUBE_ENV_LIVE_STAGING_TOKEN}
+            kubectl config set-context ${KUBE_ENV_LIVE_STAGING_NAME} --cluster=${KUBE_ENV_LIVE_STAGING_NAME} --user=circleci --namespace=${KUBE_ENV_LIVE_STAGING_NAMESPACE}
+            kubectl config use-context ${KUBE_ENV_LIVE_STAGING_NAME}
+            kubectl config current-context
+            kubectl --namespace=${KUBE_ENV_LIVE_STAGING_NAMESPACE} get pods
```

Comment:
No need for these 2 lines, it is just debug from the cloud-platforms example.

Reply:
Noted, I will make changes and create a new fresh PR.